arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "ServiceBusManagementClient",
     "description": "Azure Service Bus client",
-    "version": "2014-09-01"
+    "version": "2015-08-01"
```

Comment:
first copy the file from the old api-version and commit it. Then in the next commit have all your new api-version changes. This makes it easy for us to review things as we can see the diff in the PR.